reports/usage-analytics/Skuid_Usage.xml

```diff
@@ -0,0 +1,77 @@
+<skuidpage unsavedchangeswarning="yes" personalizationmode="server" showsidebar="true" showheader="true">
+	<models>
+<model id="Skuid_Usage" limit="1" query="false" createrowifnonefound="false" datasource="salesforce" sobject="Skuid_Usage__c">
+	<fields>
+		<field id="Name"/>
+		<field id="CreatedById"/>
+		<field id="CreatedBy.Name"/>
+		<field id="Tracked_Action_Name__c"/>
+		<field id="CreatedDate"/>
+		<field id="Tracked_Action_Category__c"/>
+	</fields>
+	<conditions/>
+	<actions/>
+</model>
+</models>
+	<components>
+<pageregion uniqueid="sk-2VbD-728" label="MainPageRegion">
+	<styles>
+		<styleitem type="background" bgtype="none"/>
+	</styles>
+</pageregion>
+</components>
+	<resources>
+		<labels/>
+		<javascript/>
+		<css/>
+		<actionsequences uniqueid="sk-2Ucx-113">
+<actionsequence id="e59f3d98-eea8-4f56-aaad-ba0aa184f30e" label="Usage Snapshot" type="reusable" uniqueid="sk-2VEO-142">
+	<description>{{$Input.trackedActionCategory}}</description>
+	<actions>
+		<action type="emptyModelData">
+			<models>
+				<model>Skuid_Usage</model>
+			</models>
+		</action>
+		<action type="createRow" model="Skuid_Usage" appendorprepend="prepend" defaultmodefornewitems="edit" affectedrows="context">
+			<models>
+				<model>Skuid_Usage</model>
+			</models>
+			<defaults>
+				<default valuesource="param" field="Name" enclosevalueinquotes="true" value="page"/>
+				<default valuesource="fieldvalue" field="Tracked_Action_Category__c" enclosevalueinquotes="true" value="{{$Input.trackedActionCategory}}"/>
+				<default valuesource="fieldvalue" field="Tracked_Action_Name__c" enclosevalueinquotes="true" value="{{$Input.trackedActionName}}"/>
+				<default valuesource="userinfo" field="CreatedById" fieldtargetobjects="User" enclosevalueinquotes="true" userinfotype="userid"/>
```

Comment:
As in my comments below, these two <default> nodes for populating CreatedById and CreatedDate should be removed, as they are unnecessary and will do nothing.